app/assets/javascripts/client/inventories/manageInventoryPermissionsLink.js

```diff
@@ -0,0 +1,18 @@
+(function () {
+  'use strict';
+  angular.module('PDRClient')
+      .directive('manageInventoryPermissionsLink', manageInventoryPermissionsLink);
+
+  function manageInventoryPermissionsLink() {
+    return {
+      restrict: 'E',
+      scope: {
+        inventoryId: '=inventoryId'
```

Comment:
I know we had touched on this briefly before, but is there ever a case in which we want to use this directive outside the context of an inventory?  That is, would `$stateParams` serve our purposes better as opposed to us having to pass that state in through the directive?